ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -99,6 +99,15 @@ then
 	#then
 	#fi
 
+	if ver hwcmp AEROFC_V1
+	then
+		if param compare SYS_AUTOSTART 0
+		then
+			param set SYS_AUTOCONFIG 1
+			param set SYS_AUTOSTART 4070
+		fi
+	fi
```

Comment:
Depends on what you want. If we assume most people will start with an RTF, I think this is ok. Otherwise we can remove this, so that the user first has to set the correct airframe.